lib/src/rules.dart

```diff
@@ -74,10 +76,12 @@ final Registry ruleRegistry = new Registry()
   ..register(new EmptyStatements())
   ..register(new HashAndEquals())
   ..register(new ImplementationImports())
+  ..register(new InvariantBoolean())
```

Comment:
Nit: in keeping with naming elsewhere, let's rename this to: `InvariantBooleans`.